btree/btree.c

```diff
@@ -2298,8 +2288,8 @@ static void generic_move(struct nd *src, struct nd *tgt,
 	}
 	node_fix(src, tx);
 	node_fix(tgt, tx);
-	mem_update(src, tx, src, sizeof *src);
-	mem_update(tgt, tx, tgt, sizeof *tgt);
+	//m0_be_tx_capture();
```

Comment:
These captures need to be a bit smarter where only the modified memory from both the nodes should be captured. This might mean a few more local variables need to be added to identify the locations in the node which have changed.

Reply:
Yes.